course_catalog/task_helpers.py

```diff
@@ -535,73 +541,79 @@ def get_micromasters_data():
     return requests.get(settings.MICROMASTERS_COURSE_URL).json()
 
 
-def parse_bootcamp_json_data(course_data, force_overwrite=False):
+def parse_bootcamp_json_data(bootcamp_data, force_overwrite=False):
```

Comment:
To follow up on my write for the new issue #2041, I think what we want here is actually two copies of this method:

- One that writes to `Course`
- One that writes to `Bootcamp`

And update the caller of `parse_bootcamp_json_data` to call both of those operations.

BYou'll almost definitely want to pass each of these methods a `copy.deepcopy` of the original data since the implementation is mutating the passed object via `pop()` and friends.

Reply:
Added a call to each function. The functions duplicate a ton of code but one will be deleted in a subsequent PR, and I am working on a refactor to the whole task_helpers file which will be another PR.